migrations/versions/be9889a7cead_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
+
+Revision ID: be9889a7cead
+Revises: e3caa0f2a16c
+Create Date: 2019-02-01 17:36:54.810899
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = 'be9889a7cead'
+down_revision = 'e3caa0f2a16c'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('events', sa.Column('is_featured', sa.Boolean(), nullable=True))
```

Comment:
nullable should be False

Reply:
@iamareebjamal There must be events which are already present who fall in neither `True` nor `False` category. I think, for those existing events, `nullable=True` seems valid. @CosmicCoder96 Pointed it out in the last PR

Reply:
No, it doesn't. Do you want frontend and android to handle a case where boolean can be null? In JS, it is the same thing, on Android, it can crash the app. An event is either featured or it is not. There is no quantum state or Schrodinger's cat involved

Reply:
The new column should be of non-null boolean type with false as default

Reply:
@iamareebjamal  I agree, nullable shouldn't be a third state, but is there anyway we can add a column in the existing events then? I mean does the migration already does that job for us?

Reply:
Yes it does if it is written correctly with a default value. Read documentation of alembic.

In pure SQL, it is just this query

```sql
alter table event add column is_featured boolean not null default false;
```

So, it is possible

Reply:
@iamareebjamal Alright, I will try and do it